package.json

```diff
@@ -54,8 +54,14 @@
     "maplibre-gl": "^4.0.2",
     "read-excel-file": "^5.7.1",
     "route-snapper": "^0.3.0",
+    "route-snapper-ts": "^0.0.3",
     "svelte": "^4.2.10",
     "svelte-maplibre": "^0.9.2",
     "uuid": "^9.0.1"
+  },
+  "overrides": {
```

Comment:
There's some slight trickery here -- the new -ts package depends on the latest 0.4.0 of the WASM package, but that has a breaking change to the binary file format. We don't have any need of that new version yet, and there's no reason to regenerate all the files right now.